tests/framework/validators/RangeValidatorTest.php

```
@@ -147,4 +148,44 @@ public function testValidateSubsetArrayable(): void
         ]);
         $this->assertTrue($val->validate('a'));
     }
+
+    public function testValidateClosureRange(): void
+    {
+        $val = new RangeValidator(['range' => function ($model, $attr) {
+            return [1, 2];
+        }]);
+        $m = new DynamicModel(['attr' => 1]);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $m. Configured minimum length is 3. (undefined)

(ShortVariable)